lib/ace/ext/papeeria/snippets/equation_snippets.coffee

```diff
@@ -0,0 +1,130 @@
+define((require, exports, module) ->
+  equationEnvironments = [
```

Comment:
These array look like constants, so please rename them in constants style: `EQUATION_ENVIRONMENTS` etc